docs/packages/pkg/QTPropertyEditor.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    QTPropertyEditor
+
+.. index::
+  single: unsorted ; QTPropertyEditor
+
+.. _pkg.QTPropertyEditor:
+
+QTPropertyEditor
+================
+
+-  `Official <https://github.com/Energid/propertyeditor>`__
+-  `Hunterized <https://github.com/hunter-packages/QTPropertyEditor>`__
```

Comment:
There is no such repository

Reply:
Could you create it?

Reply:
> Could you create it?

It's better to ask it before submitting pull request:

```
If you want to create new fork let me know about it in a corresponding issue with “new package” label
```
Documentation: https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
Installed header is `propertyeditor.h` and C++ namespace is `PropertyEditor`:
* https://github.com/Energid/propertyeditor/blob/master/lib/propertyeditor.h

Should we name the package `propertyeditor`?

Reply:
We must use QTPropertyEditor to be clear from the name of the package what it belongs to and not to be confused

Reply:
> We must use QTPropertyEditor to be clear from the name of the package what it belongs to and not to be confused

I'm not sure I understand what you mean...

Reply:
> propertyeditor it is like common name, so better use it with QT

Like I said it's already `propertyeditor.h` header and already `PropertyEditor` C++ namespace. In my opinion it's more confusing when all of that doesn't match with each other...

Anyway it's up to you, created: https://github.com/hunter-packages/QtPropertyEditor

Reply:
thanks